### PR TITLE
LVRUBY-119: Send message after comment creation

### PR DESCRIPTION
**Pull request by kmandziy**
## JIRA ticket

https://ssu-jira.softserveinc.com/browse/LVRUBY-119

## Code reviewers

- [ ] @omaik 
- [x] @AbdulRustam 

## Summary of issue

- Send message after comment creation

## Summary of change

- Added method send email to comments_controller.rb

## Review checklist

- [ ] Rubocop doesn't raise any warnings and offenses
- [ ] New features are covered with new specs
- [ ] All specs pass

### CODE REVIEW

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.68%** covered at **4.39** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.68%** covered at **4.39** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.72%** covered at **4.35** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **94.89%** covered at **4.14** hits/line

File: app/mailers/messages_mailer.rb [hunk 1]
```diff
@@ -14,6 +14,18 @@ def approve_email(user, emailchange)
     @user = user
     @token = emailchange.token
     @new_email = emailchange.email
-    mail(from: 'soft.darkside@gmail.com', to: @user.email, subject: 'You change email')
+    mail(from: Settings.app_email, to: @user.email, subject: 'You change email')
+  end
+
+  def comment_email(commentable, commenter, url)
+    @commentable = commentable
+    @user = commentable.user
```
Comment (omaik) [inline app/mailers/messages_mailer.rb]: You do not use this in view so there is no need of it to be an instance variable.

Comment (kmandziy) [reply app/mailers/messages_mailer.rb]: Already removed

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.16%** covered at **4.15** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.16%** covered at **4.15** hits/line

**Comment by vikabelous** [comment]
**Code Coverage:**
:green_heart: **95.16%** covered at **4.15** hits/line

**Pull request merged**